plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/edit/MySQLTableManager.java

```diff
@@ -115,25 +115,26 @@ protected void appendTableModifiers(DBRProgressMonitor monitor, MySQLTableBase t
             try {
                 final MySQLDataSource dataSource = table.getDataSource();
                 final MySQLTable.AdditionalInfo additionalInfo = table.getAdditionalInfo(monitor);
-                if ((!table.isPersisted() || tableProps.getProperty("engine") != null) && additionalInfo.getEngine() != null) { //$NON-NLS-1$
+                if ((!table.isPersisted() || tableProps.getProperty("additionalInfo.engine") != null) && additionalInfo.getEngine() != null) { //$NON-NLS-1$
```

Comment:
The commit that changes property names is reverted. As we discussed, please return it back.